man/man8/zpool.8

```diff
@@ -1736,13 +1781,146 @@ Verbose statistics. Reports usage statistics for individual \fIvdevs\fR within t
 .rt
 Omit statistics since boot.  Normally the first line of output reports the statistics since boot.  This option suppresses that first line of output.
 .RE
+.sp
+.ne 2
+.mk
+.na
+\fB\fB-G\fR\fR
+.ad
+.RS 12n
+.rt
+Display latency histograms:
 
+.sp
+.ne 2
+.mk
+.na
+total r/w:
+.ad
+.RS 20n
+.rt
+Total IO time (queuing + disk access).
+.RE
+.ne 2
+.mk
+.na
+disk r/w:
+.ad
+.RS 20n
+.rt
+Disk IO time (amount of time accessing physical medium).
+.RE
+.ne 2
+.mk
+.na
+sync_queue r/w:
+.ad
+.RS 20n
+.rt
+Amount of time IO spent in synchronous priority queues.
```

Comment:
Correct, sync_queue does not include disk time, only time in queue.  I'll add a line to make it clearer.